### PR TITLE
Admin can create information snippets

### PR DESCRIPTION
**Pull request by 1CIM**
### Userstory
```
As an admin
In order to put out new information on the site
I need a form where i can write new information
```

### Requested changes
- Add create call to Information modal
- Add InformationCreation view

### Media
https://recordit.co/3TrFZ8pMMx

### CODE REVIEW

File: src/views/InformationCreation.jsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+          data-cy='header'
+          variant='outlined'
+          fullWidth
+          required
+          id='standard-required'
+          label='Header'
+          type='string'
+          name='header'
+          onChange={handleChange}
+        />
```
Comment (SevaDer14) [inline src/views/InformationCreation.jsx:98]: I usually also add attribute `value={state value that is changed onChange}` to the inputs. I am not sure if it is necessary here. Did you check that it sends correct object with all the values to the API?

**Review by SevaDer14 — CHANGES_REQUESTED** [review]
need to change the route for info creation. And some question

File: cypress/integration/adminCanCreateInfoSnippet.feature.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        beforeEach(() => {
+          cy.intercept('POST', '**/api/information/**', {
+            statusCode: 400,
+            body: {
+              error_message: 'An error occurred',
+            },
+          })
+        })
+
+        context('unsuccessfully, by clicking `submit` btn', () => {
```
[the remark this answers is not in the document]
Comment (1CIM) [reply cypress/integration/adminCanCreateInfoSnippet.feature.js:52]: removed

File: src/modules/Information.js [hunk 3]
```diff
@@ -28,6 +28,23 @@ const Information = {
     }
   },
 
+  async create(info) {
+    try {
+      let response = await axios.post(
+        'api/information/create',
```
[the remark this answers is not in the document]
Comment (1CIM) [reply src/modules/Information.js]: fixed 👌 for article as well

File: src/views/InformationCreation.jsx (hunk 1 above)
Comment (1CIM) [reply src/views/InformationCreation.jsx]: was not needed but had to to add a different on change handler (handleChecked)

**Review by SevaDer14 — APPROVED** [review]

**Pull request merged**